relays/bin-substrate/src/cli/bridge.rs

```diff
@@ -97,7 +126,7 @@ impl CliBridgeBase for MillauToRialtoCliBridge {
 	type Target = relay_rialto_client::Rialto;
 }
 
-impl HeadersCliBridge for MillauToRialtoCliBridge {
+impl RelayHeadersCliBridge for MillauToRialtoCliBridge {
```

Comment:
Id also consider those chain-specific structs to the `chains` folder - imo there are too much structs here and the number will grow with a time. WDYT? If you agree, we may do that in a follow-up PRs

Reply:
Sounds very good. At first sight it doesn't seem to imply many changes so I can try to do it in this PR.

Reply:
Actually I'll do this in a separate PR. This PR is getting too complex.